src/CreateGuesser.tsx

```diff
@@ -1,9 +1,12 @@
+import type { PropsWithChildren, ReactNode } from 'react';
```

Comment:
Could you add it after the "true" import?

Reply:
OK